substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -734,6 +734,7 @@ def _debuginfotest(native_image, path, build_only, args):
                          '-cp', classpath('com.oracle.svm.test'),
                          '-Dgraal.LogFile=graal.log',
                          '-g',
+                         '-H:-SpawnIsolates',
```

Comment:
That's a good idea but probably best left to a follow-up issue.

Reply:
At https://github.com/oracle/graal/pull/3046/files#diff-eab1ebfe7b30eac56f68f32f6eff8466a51b176a60fcd6b0309d974f48579057L744 please add something like
```
    env = environ.copy()
    if 'GDB_BIN' not in env:
        env['GDB_BIN'] = 'gdb'
```
and then use `env['GDB_BIN']` instead of hardcoding `'gdb'` in `mx.run`.

Reply:
Done.